k8s/src/main/scala/io/buoyant/k8s/Watchable.scala

```diff
@@ -21,6 +22,8 @@ private[k8s] abstract class Watchable[O <: KubeObject: TypeReference, W <: Watch
 
   protected def backoffs: Stream[Duration]
   protected def stats: StatsReceiver
+  protected implicit val timer = HighResTimer.Default
```

Comment:
is HighResTimer required? is this just the default recommended thing? (used to seeing DefaultTimer.twitter or whatever it was)